main.py

```diff
@@ -0,0 +1,4 @@
+eleks = ['Dean', 'Sam', 'Ben', 'David']
+toshiba = eleks.copy()
```

Comment:
and where is toshiba? ACCORDING TO TASK : You have 2 companies with people. and than eleks was taken by toshiba